docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -436,7 +436,23 @@ The connector supports pushdown for a number of operations:
 
 .. include:: join-pushdown-enabled-true.fragment
 
-.. include:: no-pushdown-text-type.fragment
+Predicate pushdown support
+""""""""""""""""""""""""""
+
+The connector supports pushdown of any predicates on ``VARCHAR`` and
+``NVARCHAR`` columns. The following operators are supported:
+
+- EQUAL
```

Comment:
I dont think we use that in others .. it probably more like =, !=  .. check with others